OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -158,37 +174,55 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </dimensions>
                                 </layout>
                             </layoutDesc>
-                            
+
                         </objectDesc>
-                        
+
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+
                             </handNote>
                         </handDesc>
-                        
+
                        <!-- <decoDesc>
-                         
+
                         </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                            </binding>
-                        </bindingDesc>-->
-                        
+
+                       <bindingDesc>
+                         <binding contemporary="true Ethiopian" xml:id="binding">
+                             <decoNote xml:id="b1" type="bindingMaterial">
+                               The manuscript is bound in <material key="wood">wooden</material> boards and covered with <material key="leather"/> on the spine.
+                             </decoNote>
+                             <decoNote xml:id="b2" type="Boards">
+                               The boards are cut to the same size as the bookblock and have shallow external bevels. The grain runs vertically.
+                               The external surface of the boards is polished, while the inner surface exhibits signs of the working tool.
+                             </decoNote>
+                             <decoNote xml:id="b3" type="SewingStations">4</decoNote>
+                             <decoNote xml:id="b4">
+                               The sewing is a chainstitch on four sewing stations, broken and repaired with a different kind of thread. Both threads are Z-plied.
+                              </decoNote>
+                             <decoNote xml:id="b5" type="Spine">
+                               The small holes close to the head and tail of the quires probably indicate that quire tackets were originally present.
+                               The spine has been protected by a sort of spine lining made by joining together a brown and a white piece of leather.
+                             </decoNote>
+                             <decoNote xml:id="b6" type="SlipCase">
+                               The original leather case is preserved. It is a one-piece, double-walled satchel where the back panel overlaps over the front.
```

Comment:
Here, "leather case" and "satchel" are used, above "slip case". To us, it is clear, but I wonder whether this might be confusing to those who don't know these cases well. Maybe here "leather slip case" could be used in the first sentence?